plugins/app-backend/src/service/router.ts

```diff
@@ -21,6 +21,7 @@ import { Logger } from 'winston';
 import { notFoundHandler, resolvePackagePath } from '@backstage/backend-common';
 import { Config } from '@backstage/config';
 import { injectConfig, readConfigs } from '../lib/config';
+import fs from 'fs';
```

Comment:
Use `pathExists` from `fs-extra`!

Reply:
Isn't that just a wrapper around `access`? https://github.com/jprichardson/node-fs-extra/blob/5b29ae3aa5198c33820a07eaa4493a5b6d01fa82/lib/path-exists/index.js#L5-L7

Reply:
But it looks nicer 😉

Reply:
May be functionally the same, but we generally use `fs-extra` instead of `fs.promises`

Reply:
Done, fs-extra was available in the plugin anyway 👍